lib/views/tree-view.js

```diff
@@ -234,6 +234,93 @@ class TreeView extends ScrollView {
     this.getTitle = () => 'Remote-FTP';
   }
 
+	TreeView.prototype.handleEvents = function() {
```

Comment:
Please use ES6 code style / guide. Install [eslint](https://atom.io/packages/linter-eslint) to write a bug-free code to Atom.

Error: `Parsing error: Unexpected token . (fatal)`